actors/miner/src/lib.rs

```diff
@@ -4854,147 +4856,41 @@ fn balance_invariants_broken(e: Error) -> ActorError {
 }
 
 impl ActorCode for Actor {
-    fn invoke_method<RT>(
-        rt: &mut RT,
-        method: MethodNum,
-        params: &RawBytes,
-    ) -> Result<RawBytes, ActorError>
-    where
-        RT: Runtime,
-        RT::Blockstore: Clone,
```

Comment:
How are we dealing with this?

Reply:
The [dispatch macro slaps `Clone` on](https://github.com/filecoin-project/builtin-actors/blob/065286fc90c4f6663955b568fe90b7c61b75d61a/runtime/src/dispatch.rs#L36) for all `ActorCode::invoke_method()`, since [the EVM actor needs it too].(https://github.com/filecoin-project/builtin-actors/blob/next/actors/evm/src/lib.rs#L266).